spec/system/visit_user_journey_confirmation_page_spec.rb

```diff
@@ -84,4 +84,55 @@
       expect(page).to have_content 'Because your service provider does not support dual running'
     end
   end
+
```

Comment:
If it's not too much pain; you might want to change  ReplaceEncryptionCertificateEvent.create to factoryBot the `factory :replace_encryption_certificate_event` for the

Reply:
@jkunle thanks. I added a new commit that converts the event creation to be done using factories. It also removes them the `before` block and sets them individually for each spec as `before` was causing events to be mass applied unnecessarily to `it` blocks. Also if I commented it out, then that made no difference and the tests still passed so I changed it to navigate through the event/certificate relation to show that it was actually being utilised.